src/data/tools.json

```
@@ -1366,6 +1366,19 @@
     "description": "The open-source Shopify alternative.",
     "logo": "medusa.png",
     "tag": [ "development platform", "tool", "API", "open source"]
+  },
+  {
+  "id": 151,
```

Note that the `id` needs to be incremented by 1

```suggestion
  "id": 152,
```